passes/memory/memory_collect.cc

```diff
@@ -218,6 +218,10 @@ Cell *handle_memory(Module *module, RTLIL::Memory *memory)
 	mem->setPort("\\RD_DATA", sig_rd_data);
 	mem->setPort("\\RD_EN", sig_rd_en);
 
+	// Copy attributes from RTLIL memory to $mem
```

Comment:
This looks to be from #1501, which has already been merged into master. Do you know why it's appearing here again?